pipelines/nextflow/workflows/dumper_pipeline/main.nf

```diff
@@ -21,6 +21,15 @@ params.brc_mode = 0
 params.dbname_re = ''
 params.output_dir = './dumper_output'
 params.password = ''
+params.cache_dir = './cache'
```

Comment:
I'm personally more inclined to use either `params.workDir` or something similar rather than `.` by default, as it makes it fixed regardless of where you are running the pipeline from (applicable to all params, not just this one).